lnbits/extension_manager.py

```diff
@@ -51,13 +51,14 @@ def from_installable_ext(cls, ext_info: "InstallableExtension") -> "Extension":
         )
 
 
+# All subdirectories in the current directory, not recursive.
+
 class ExtensionManager:
     def __init__(self, include_disabled_exts=False):
         self._disabled: List[str] = settings.lnbits_disabled_extensions
         self._admin_only: List[str] = settings.lnbits_admin_extensions
-        self._extension_folders: List[str] = [
-            x[1] for x in os.walk(os.path.join(settings.lnbits_path, "extensions"))
-        ][0]
+        p = Path(settings.lnbits_path, "extensions")
```

Comment:
- what happens if `extensions` dir does not exits?

Reply:
Should still have a check!